src/ResourceManager/Dns/Commands.Dns/Records/NewAzureDnsRecordSet.cs

```diff
@@ -25,34 +25,44 @@ namespace Microsoft.Azure.Commands.Dns
     /// <summary>
     /// Creates a new record set.
     /// </summary>
-    [Cmdlet("New", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "DnsRecordSet", SupportsShouldProcess = true),OutputType(typeof(DnsRecordSet))]
+    [Cmdlet("New", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "DnsRecordSet", DefaultParameterSetName = "Fields", SupportsShouldProcess = true),OutputType(typeof(DnsRecordSet))]
```

Comment:
can you update the help file for the cmdlet as well : 

https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Dns/Commands.Dns/help/New-AzureRmDnsRecordSet.md

Heres the documentation for help generation : https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md

Reply:
updated.